packages/tools/examples/petCt/index.ts

```diff
@@ -698,7 +705,119 @@ async function setUpDisplay() {
   // Render the viewports
   renderingEngine.render();
 }
+addButtonToToolbar({
+  title: 'toggle volume3d',
+  onClick: async () => {
+    const ptImageIds = await getPtImageIds();
+    // Define a volume in memory
+    const volume = await volumeLoader.createAndCacheVolume(ptVolumeId, {
+      imageIds: ptImageIds,
+    });
+    volume.load();
+
+    const viewportInput = {
+      viewportId: viewportIds.PETMIP.CORONAL,
+      type: ViewportType.VOLUME_3D,
+      element: element_mip,
+      defaultOptions: {
+        orientation: Enums.OrientationAxis.CORONAL,
+        background: <Types.Point3>[1, 1, 1],
+      },
+    };
+    renderingEngine.enableElement(viewportInput);
+
+    const viewport = <Types.IVolumeViewport>(
+      renderingEngine.getViewport(viewportIds.PETMIP.CORONAL)
+    );
+
+    const ptVolumeDimensions = volume.dimensions;
+
+    // Only make the MIP as large as it needs to be.
+    const slabThickness = Math.sqrt(
+      ptVolumeDimensions[0] * ptVolumeDimensions[0] +
+        ptVolumeDimensions[1] * ptVolumeDimensions[1] +
+        ptVolumeDimensions[2] * ptVolumeDimensions[2]
+    );
+
+    viewport.setVolumes([
+      {
+        volumeId: ptVolumeId,
+        callback: setPetTransferFunctionForVolumeActor,
+        slabThickness: slabThickness,
+        blendMode: BlendModes.MAXIMUM_INTENSITY_BLEND,
+      },
+    ]);
 
+    const ptVoiSynchronizerId = 'PT_VOI_SYNCHRONIZER_ID2';
+    const ptVoiSynchronizer = createVOISynchronizer(ptVoiSynchronizerId);
+
+    [
+      viewportIds.PT.AXIAL,
+      viewportIds.PT.SAGITTAL,
+      viewportIds.PT.CORONAL,
+      viewportIds.FUSION.AXIAL,
+      viewportIds.FUSION.SAGITTAL,
+      viewportIds.FUSION.CORONAL,
+      viewportIds.PETMIP.CORONAL,
```

Comment:
Also I don't think you need to redefine the toolgroups, just grab the existing toolGroup for the MIP and modify its viewport (I don't think that is even needed, maybe just re-enable it?)